### PR TITLE
Improve data structure to not break on reconnections

### PR DESCRIPTION
**Pull request by Tainan404**
### What does this PR do?
This PR changes how the data is registered and stored on the client side.
Now all subscriptions requested from the client are exported as local collections.
The data synchronization is handled by LiveDataEventBroker, avoiding that these collections get empty on a reconnect.
It also contains enhancements to improve the UX during a reconnection.

### CODE REVIEW

File: bigbluebutton-html5/client/main.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@
 */
 /* eslint no-unused-vars: 0 */
 
-import '../imports/ui/services/collection-hooks/collection-hooks';
+import './wdyr';
```
Comment (antobinary) [inline bigbluebutton-html5/client/main.jsx:20]:
```suggestion
```

This package was removed

File: bigbluebutton-html5/private/config/settings.yml [hunk 2]
```diff
@@ -191,7 +191,7 @@ public:
   externalVideoPlayer:
     enabled: true
   kurento:
-    wsUrl: HOST
+    wsUrl: wss://tainan.bbbvm.imdt.com.br/bbb-webrtc-sfu
```
Comment (antobinary) [inline bigbluebutton-html5/private/config/settings.yml:194]:
```suggestion
    wsUrl: HOST
```

File: bigbluebutton-html5/imports/ui/components/subscriptions/component.jsx [hunk 3]
```diff
@@ -22,7 +25,7 @@ const SUBSCRIPTIONS = [
   'voiceUsers', 'whiteboard-multi-user', 'screenshare', 'group-chat',
   'presentation-pods', 'users-settings', 'guestUser', 'users-infos', 'note', 'meeting-time-remaining',
   'local-settings', 'users-typing', 'record-meetings', 'video-streams',
-  'connection-status', 'voice-call-states', 'external-video-meetings', 'breakouts',
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/subscriptions/component.jsx:25]: Where did `'voice-call-states'` go?

File: bigbluebutton-html5/imports/ui/components/app/container.jsx [hunk 4]
```diff
@@ -119,10 +119,6 @@ const currentUserEmoji = (currentUser) => (currentUser
 );
 
 export default injectIntl(withModalMounter(withTracker(({ intl, baseControls }) => {
-  if (Auth.connectionID !== Meteor.connection._lastSessionId) {
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/app/container.jsx:122]: @Tainan404 could you please explain briefly why this is safe to remove?

Comment (Tainan404) [reply bigbluebutton-html5/imports/ui/components/app/container.jsx:122]: In the current PR, we are able to keep the client working offline and on the reconnection we re-authenticate the user, so, the connection changes before user be re-authed causing the user to be kicked out from the meeting

**Comment by jfsiebel** [comment]
After a user is promoted from `VIEWER` to `MODERATOR` an error happens:

image

**Review by jfsiebel — APPROVED** [review]

**Review by antonbsa — APPROVED** [review]
After running the `Trigger: Meteor Reconnection` test several times in a row (~ 30-50), I got no failures!

**Pull request merged**